### ISSUE TITLE
manifest option in dependencies is ignored 

### ISSUE THREAD

**Issue by alexissellier** [okteto/okteto#2825]
**Describe the bug**

The `manifest` option inside the declaration of a dependency is ignored and therefore only the manifest at the base folder is used.

**To Reproduce**
With this manifest
```
dependencies:
  grafana:
    repository: https://github.com/ankorstore/poc-okteto-dependencies
    manifest: okteto.yml
    branch: monitoring
    wait: true
  base:
    repository: https://github.com/ankorstore/poc-okteto-dependencies
    manifest: ./base/okteto.yml
    branch: monitoring
    wait: true
```
The `./base` is ignored and my dependency is using the manifest at the base directory.

This is the content of the configmap in my namespace 

```
$ k get cm/okteto-git-base -o yaml
apiVersion: v1
data:
  actionName: 6249acfd-5cd8-4455-a3dc-8a38ac3b0b76
  branch: monitoring
  commitSha: ""
  fallbackRepository: ssh://git@github.com/ankorstore/poc-okteto-dependencies
  filename: ""
  githubCommentId: ""
  icon: ""
  name: base
```
```
$ k get cm/okteto-git-grafana -o yaml
apiVersion: v1
data:
  actionName: 8e9db9a1-f441-4599-891a-25dbade57f66
  branch: monitoring
  commitSha: ""
  fallbackRepository: ssh://git@github.com/ankorstore/poc-okteto-dependencies
  filename: ""
  githubCommentId: ""
  icon: ""
  name: grafana

```

**Expected behavior**
The full path is used to fetch my manifest

**Desktop:**

- OS: Arch Linux
- Okteto cli version: okteto version 2.3.2 
- Okteto entreprise chart version :  helm.sh/chart: okteto-enterprise-0.13.0
- Application verison 1.12.132-4aadd0bc

**Comment by pchico83** [comment]
@alexissellier while we investigate the issue, could you check if the repo is properly deploying using:
```
okteto pipeline deploy \
    -r repository: https://github.com/ankorstore/poc-okteto-dependencies \
    -f base/okteto.yml \
    --branch monitoring \
    --wait
```

**Comment by teresaromero** [comment]
hi @alexissellier when specifying a path that is not existent, the dependency pipeline fallback to the `okteto.yml` at the root of the repo. Could you check that the `base` manifest path is pointing to an existent file in that branch? 
I've only been able to reproduce this when the manifest field at the dependency is not a valid file. When this happens the filename field at the configmap is empty.

**Comment by alexissellier** [comment]
Hi, I saw your questions and I will take some time tomorrow to test that

**Comment by pchico83** [comment]
@alexissellier we have identified the issue and we are working on a fix. I will update the issue when we have an ETA

**Comment by teresaromero** [comment]
@alexissellier we released a fix at Cloud and 2.4.2 CLI version. Please update your CLI and check with Cloud if this is still happening. Thnks

**Comment by alexissellier** [comment]
Apologies for the delay. I've tested it and it works fine with version 2.4.2.
